### PR TITLE
feat:add show warnings

### PR DESCRIPTION
**Pull request by wang1309**
**What this PR does**: add show warnings

**Which issue(s) this PR fixes**:
ref #360 

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/ast/show.go [hunk 1]
```diff
@@ -456,3 +457,27 @@ func (s *ShowTableStatus) Restore(flag RestoreFlag, sb *strings.Builder, args *[
...
+	Limit *LimitNode
+}
+
+func (s *ShowWarnings) Validate() error {
+	return nil
+}
+
+func (s *ShowWarnings) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
+	sb.WriteString("SHOW ")
+	sb.WriteString(" WARNINGS ")
```
Comment (jjeffcaii) [inline pkg/runtime/ast/show.go:472]: Merge

File: pkg/runtime/plan/dal/show_warnings.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	"context"
+)
+
+import (
+	"github.com/arana-db/arana/pkg/proto"
+	"github.com/arana-db/arana/pkg/proto/rule"
+	"github.com/arana-db/arana/pkg/runtime/ast"
+	"github.com/arana-db/arana/pkg/runtime/plan"
+	"github.com/pkg/errors"
+	"strings"
```
Comment (jjeffcaii) [inline pkg/runtime/plan/dal/show_warnings.go:30]: reformat

**Review by jjeffcaii — APPROVED** [review]
LGTM

File: pkg/runtime/ast/show.go (hunk 1 above)
Comment (wang1309) [reply pkg/runtime/ast/show.go]: done

File: pkg/runtime/plan/dal/show_warnings.go (hunk 2 above)
Comment (wang1309) [reply pkg/runtime/plan/dal/show_warnings.go]: done

**Review by dongzl — APPROVED** [review]

**Review by Mulavar — APPROVED** [review]

**Pull request merged**